### PR TITLE
1 setup and document frontend development environment

### PR DESCRIPTION
**Pull request by tuomasmustakallio**
Initialized next.js application with docker. Added setup guide to README.md

### CODE REVIEW

File: client/package-lock.json
Comment (EddieTheCubeHead) [inline client/package-lock.json:1]: Please add package-lock to .gitignore and remove the file.

Sometimes getting git to remove a file added to gitignore is a bit finicky so I recommend doing as follows:
 - Delete package-lock
 - Commit and push
 - Add package-lock to gitignore
 - Run npm install again locally to restore your local package-lock copy

File: client/public/next.svg
Comment (EddieTheCubeHead) [inline client/public/next.svg:1]: Do we need these two svg files? If not can we remove them?

File: client/public/vercel.svg
Comment (EddieTheCubeHead) [inline client/public/vercel.svg:1]: Do we need these two svg files? If not can we remove them?

**Review by EddieTheCubeHead — CHANGES_REQUESTED** [review]

**Review by ddevarsh7 — APPROVED** [review]

File: client/app/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import type { Metadata } from "next";
+import { Inter } from "next/font/google";
+import "./globals.css";
+
+const inter = Inter({ subsets: ["latin"] });
+
+export const metadata: Metadata = {
+  title: "Create Next App",
+  description: "Generated by create next app",
```
Comment (EddieTheCubeHead) [inline client/app/layout.tsx:9]: Sorry for leaving extra comments after the first review:

Can we already set title to "Stagnum" and description to "A pool-based playback handler for Spotify"?

**Review by EddieTheCubeHead — APPROVED** [review]

**Review by OttoAhlfors — APPROVED** [review]

**Pull request merged**